_includes/v19.2/sql/movr-statements-no-data.md

```diff
@@ -0,0 +1,10 @@
+The following examples use MovR, a fictional vehicle-sharing application, to demonstrate CockroachDB SQL statements. For more information about the MovR example application, see [MovR: A Global Ride-sharing App](movr.html).
```

Comment:
I think we should standardize on either ride-sharing or vehicle-sharing.  AFAICT it's really more "vehicle sharing" since there is no drivers table, etc.  Is that correct?

Reply:
I 100% agree on this. It is, in fact, a vehicle-sharing app. I confirmed this with Nate last week. I plan to make this global change a part of the PR.